app/templates/subsidy/measure-offer/steps.hbs

```diff
@@ -0,0 +1,62 @@
+{{page-title "Steps"}}
+
+
+<AuBodyContainer id="content">
+  <SidebarContainer>
+    <:sidebar>
+        {{#if this.setup.isRunning}}
+          <AuLoader />
+        {{else}}
+          {{#if this.model.activeApplicationFlow}}
+            <nav>
+              <ul class="au-c-list-navigation">
+                {{#each this.model.activeApplicationFlow.sortedDefinedSteps as |step|}}
+                  <li class="au-c-list-navigation__item">
+                    {{#if step.subsidyProceduralStep.description}}
+                        <AuLink 
+                          class="au-c-list-navigation__link"
+                          style="flex-direction: column; align-items: start;"
+                          {{!template-lint-disable no-inline-styles}}
+                        >
+                        <div class="au-u-flex au-u-flex--spaced-tiny">
+                          <AuHeading @skin="4">{{step.order}}. {{step.subsidyProceduralStep.description}}</AuHeading></div>
```

Comment:
```suggestion
                          <AuHeading @skin="4">{{step.subsidyProceduralStep.description}}</AuHeading></div>
```
The step order is not needed here after all, maybe in a future feature where we can visualize the active, and 'skipped' steps